src/drivers/boards/auav-x21/board_config.h

```diff
@@ -210,6 +210,9 @@
 
 #define BOARD_NAME "AUAV_X21"
 
+// Run the FMU task on its own thread to reduce jitter
+#define PX4FMU_TASK
```

Comment:
Minor nitpick, but it's often safer and easier to read when you define and check a specific value.

```C
    #define PX4FMU_TASK 1

    #if PX4FMU_TASK
        // px4fmu_task enabled
    #else 
        // px4fmu_task disabled
    #endif
```